qi-lib/flow/core/compiler/deforest/cps.rkt

```diff
@@ -21,7 +21,7 @@
 (define-syntax inline-compose1
   (syntax-rules ()
     [(_ f) f]
-    [(_ [op f] rest ...) (op f (inline-compose1 rest ...))]))
```

Comment:
A suggestion for d9ec8600a01093c7e34ff13446de5ac39248fa36's message: Perhaps "deforest take with boxes"?